apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -61,37 +53,23 @@ const useNoAnimationStyles = makeStyles({
     },
   },
 });
-export const TestWrapperDecoratorNoAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
+export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
   const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.root, 'testWrapper');
+  const className = mergeClasses(noAnimationStyles.paused, 'testWrapper');
   return (
     <div style={{ display: 'flex' }}>
       <div
         className={className}
         style={{
           padding: '10px',
-          overflow: 'hidden',
         }}
       >
         {story()}
       </div>
     </div>
   );
 };
-export const TestWrapperDecoratorPauseAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
+export const PauseAnimationDecorator: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
   const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.paused, 'testWrapper');
-  return (
-    <div style={{ display: 'grid' }}>
-      <div
-        className={className}
-        style={{
-          padding: '10px',
-          overflow: 'hidden',
-        }}
-      >
-        {story()}
-      </div>
-    </div>
-  );
+  return <div className={noAnimationStyles.paused}>{story()}</div>;
```

Comment:
You probably need to add `testWrapper` as it's expected from the test decorators

```suggestion
  return <div className={mergeClasses(noAnimationStyles.paused, 'testWrapper')}>{story()}</div>;
```

Reply:
This one is meant to be combined with a TestWrapperDecorator, so it should not have the testWrapper class itself.